sync-template.sh

```diff
@@ -502,14 +522,19 @@ function do_sync() {
   )
 
   for REPO in ${REPOLIST}; do
-    inform "Synchronizing ${REPO} wiht ../${LSR_TEMPLATE}."
+    inform "Synchronizing ${REPO} with ../${LSR_TEMPLATE}."
     runcmd "[[ -d \"${REPO}\" ]] && rm -rfd ${REPO} || :"
```

Comment:
Isn't this dangerous with --local ? IIUC the idea of --local is that WORKDIR is not being treated as a temporary directory.

Reply:
yes, fixed